### DISCUSSION TITLE
Any good 6 buttons device?

### DISCUSSION THREAD

**Discussion by andreapx** [Koenkk/zigbee2mqtt#18936] [Devices]
I’ve been using an Aqara 6 buttons for long time and I really like it.
I wanted to buy a couple more, but I can’t find them anywhere! On Aliexpress in the official store there is just the 2 buttons version, on other stores there is the 4 buttons one, but there no track of the 6 buttons.
Do you guys know why they retired them?
Any alternative?
Thanks!

**Comment by Westcott1** [comment]
[https://www.aliexpress.com/item/1005002219929565.html](url)
Clicking on the link doesn't seem to work, but the link itself is correct!

**Reply by andreapx** [reply]
I've been to that seller store, , but the 6 button version is not available.
The 6 version one just disappeared!

**Reply by Westcott1** [reply]
It was there when I looked this morning. and you could buy 2 of them!

**Reply by andreapx** [reply]
It's still there, but if you check better you see that the quantity is 0 and can't be changed:
![image](https://github.com/Koenkk/zigbee2mqtt/assets/13846728/4a6f0bff-80be-4a47-8127-b6ec1ad0703e)
